hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -934,15 +934,15 @@
       tests: ["chip_aon_timer_wdog_bite_reset"]
     }
     {
-      name: chip_sw_pwrmgr_all_reset_reqs
-      desc: '''Verify that the chip can be reset by ALL available reset sources.
+      name: chip_sw_pwrmgr_wdog_reset
```

Comment:
Yes, chip_aon_timer_wdog_bite_reset performs two wdog resets : 1. normal mode followed by watch dog reset, 2. deep sleep mode followed by watch dog reset

pwrmgr_sysrst_ctrl_reset performs two resets : 1. watch dog reset, 2. sysrst 
If we have other resets to be covered, we need to create new tests.

Reply:
Yes, chip_aon_timer_wdog_bite_reset performs two wdog resets : 1. normal mode followed by watch dog reset, 2. deep sleep mode followed by watch dog reset

pwrmgr_sysrst_ctrl_reset performs two resets : 1. watch dog reset, 2. sysrst
If we have other resets to be covered, we need to create new tests.

Reply:
Yes, we can extend pwrmgr_sysrst_ctrl_reset by adding new resets including the PAD reset. I think we need to list up new resets to be added as 1st step.

Reply:
it feels like to me that the sysrst_ctrl_reset test was meant to be the `all_resets` test. I think @jdonjdon specifically exercised PAD reset in deep sleep, but it probably makes sense to exercise in the all resets test as well.

In terms of all resets, we have all the hardware request sources + alert escalated reset + PAD resets. 

It might be worth walking over this part of the testplan again to clarify, I get the feeling we have some duplicates and some other gaps.